batch/batch/front_end/front_end.py

```diff
@@ -432,6 +443,15 @@
 
                 id = (batch_id, job_id)
 
+                if start_job_id is None:
+                    start_job_id = job_id
+
+                if batch_format_version > 1 and last_job_idx:
```

Comment:
I prefer previous (prev) to last.  Last is ambiguous, it can mean previous or final.